### ISSUE TITLE
Limit or Bypass Github API Restrictions

### ISSUE THREAD

**Issue by oharaandrew314** [oharaandrew314/TinkerTime#186]
While developing, I hit the unauthenticated request limit for Github pretty quickly.  I do not want to ask users to provide their credentials, so I believe that reverting to the HTML crawler is the best solution.

@MarkSRobinson, do you have any input or suggestions?

**Comment by oharaandrew314** [comment]
I just noticed that you mentioned rawgit.com in your PR.  I will check that out, too.

**Comment by oharaandrew314** [comment]
I will try expanding the html and json caching to persist between TinkerTime sessions

cached files will be invalidated after 10 minutes.

**Comment by cgruber** [comment]
I did a version of github API and hit the same limits.  With even a fraction of my use, I hit the limits during development.  It was quite frustrating, which is why I ended up not submitting a pull request for it.  I was surprised to see that someone had merged a version. :/ 

A few issues could be helped - one of which is to allow a github login screen, keep the token, so developer builds can use such... but even then, I actually think that I'm going to advocate with the github folks that certain APIs not be rate limited on an hourly quota per IP, but on a throughput basis, so that it's a little more reasonable.  X queries per minute or something.  Especially to APIs that are published information like releases, etc.  I got really high-quality information back without scraping, but the rate limiting is nuts and overzealous.
